public/app/features/explore/TraceView/createSpanLink.tsx

```diff
@@ -116,7 +115,11 @@ function legacyCreateSpanLinkFactory(
           dataLink = getLinkForSplunk(span, traceToLogsOptions, logsDataSourceSettings);
           break;
         case 'elasticsearch':
-          dataLink = getLinkForElasticsearch(span, traceToLogsOptions, logsDataSourceSettings);
+          dataLink = getLinkForElasticsearchOrOpensearch(span, traceToLogsOptions, logsDataSourceSettings);
+          break;
+        case 'grafana-opensearch-datasource':
+          dataLink = getLinkForElasticsearchOrOpensearch(span, traceToLogsOptions, logsDataSourceSettings);
```

Comment:
nit: we're setting dataLink to the same value in both cases. Can we merge them?

Reply:
you are right, but i personally don't like fall-through case-statements.. but it's your squad's codebase, so if you prefer to do it so, i can adjust it. just to clarify, is this what you'd prefer to have?:
```typescript
switch(...) {
    case 'loki':
        datalink = ...
        break;
    case 'splunk':
       datalink = ...
       break;
    case 'elasticsearch':
    case 'opensearch':
        datalink = ....
        break;
}
```
?

Reply:
Ahh this is no problem. That is what I head in mind. I'm happy with either as it's no big deal :)